src/compas_view2/shapes/collection.py

```diff
@@ -0,0 +1,25 @@
+from compas.geometry import Shape
+from compas.datastructures import Mesh
+# import numpy as np
+
+__all__ = ['Collection']
+
+
+class Collection(Shape):
+    """A collection of shapes
+    """
+
+    def __init__(self, shapes=[]):
```

Comment:
this can have unexpected side-effects. better would be
```
def __init__(self, shapes=None):
    self.shapes = shapes or []
```

Reply:
Makes sense. I'm also thinking probably `shapes` not the best name since it also supports Mesh. How about `children` ??

Reply:
i tend to use `items`